tools/tune/dcblock/example_dcblock.m

```diff
@@ -36,3 +77,54 @@ function example_dcblock()
 rmpath ./../common
 
 end
+
+% Finds with iterative search parameter R for given cutoff frequency
+function R = dcblock_rval_calculate(fs, fc_target)
+
+if (fc_target / fs < 10 / 48e3 || fc_target / fs > 1000 / 48e3)
+	error("Illegal fc_target");
+end
+
+h_target = 1 / sqrt(2); % -3.01 dB
+R = 0.5;
+R_step = 0.005;
+sign = 1;
+err_prev = inf;
+w = 2 * pi * fc_target / fs;
+j = sqrt(-1);
+z = exp(j * w);
+
+% Iteration 0
+h = (1 - z^-1) / (1 - R * z^-1);
+a = abs(h);
```

Comment:
The value assigned to `a` could be unnecessary.

Reply:
Yep, remain from previous iteration code version, both can be deleted.